my_cinema_project/cinema/views.py

```diff
@@ -0,0 +1,41 @@
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from rest_framework import status
+from django.shortcuts import get_object_or_404
+
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+
+
+
+@api_view(["GET", "POST", "PUT", "PATCH", "DELETE"])
+def movie_view(request, pk=None):
+    if request.method == "GET":
+        if pk:
+            movie = get_object_or_404(Movie, pk=pk)
+            serializer = MovieSerializer(movie)
+            return Response(serializer.data, status=status.HTTP_200_OK)
+        else:
+            movies = Movie.objects.all()
+            serializer = MovieSerializer(movies, many=True)
+            return Response(serializer.data, status=status.HTTP_200_OK)
```

Comment:
There is no need for the `else` statement, remove it. The cod will not execute after return anyway.